.github/workflows/system-tests-enterprise.yml

```diff
@@ -226,8 +231,14 @@ jobs:
       matrix:
         test_component: [api,runtimes,projects,model_monitoring,examples,backwards_compatibility,feature_store]
     steps:
+    - name: set var  branch_to_test  to development
+      if: github.event.schedule=='0 22 * * *'
```

Comment:
it is wrong to bind the schedule template *value* to the workflow (e.g.: can only check if it is not empty).
I understand you wanted to do it in order to understand what is the origin of the request, but I think there should be a different approach here. perhaps as suggested above